e2e/__tests__/jest.config.ts.test.ts

```diff
@@ -74,50 +79,73 @@ test('traverses directory tree up until it finds jest.config', () => {
   expect(summary).toMatchSnapshot();
 });
 
-const jestPath = require.resolve('jest');
-const jestTypesPath = jestPath.replace(/\.js$/, '.d.ts');
-const jestTypesExists = fs.existsSync(jestTypesPath);
+onNodeVersions('<22.6', () => {
+  const jestPath = require.resolve('jest');
+  const jestTypesPath = jestPath.replace(/\.js$/, '.d.ts');
+  const jestTypesExists = fs.existsSync(jestTypesPath);
+
+  (jestTypesExists ? test : test.skip).each([true, false])(
+    'check the config disabled (skip type check: %p)',
```

Comment:
Skipping this test in new node versions since the node feature only transpiles and doesn't type check.